reflection/reflection.fbs

```diff
@@ -45,6 +50,7 @@ table Enum {
     values:[EnumVal] (required);  // In order of their values.
     is_union:bool = false;
     underlying_type:Type (required);
+    attributes:[Attribute] (required);
```

Comment:
these should not be required either, since most things don't have attributes, you should not write this field for them if there's 0 elements.